.ci/pre-deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# load common functions
+script_path=$( cd "$(dirname "$0")" || exit 2; pwd -P )
+# shellcheck source=scripts/util.sh
+source "$script_path/util.sh"
```

Comment:
You can skip pulling in the util.sh file here because we're not running anything in parallel.